### ISSUE TITLE
[Enhancement]: Add hotkey to changing time scale and lock

### ISSUE THREAD

**Issue by alan42521** [yukieiji/UnityExplorer#41]
### Describe the new feature or enhancement

sometimes it is essential to pause the game to catch something, however unityexplorer could only pause by typing the speed(timescale) and press f7

**Comment by yukieiji** [comment]
I have no idea how to lock a Unity game, is it possible to do that?

**Comment by alan42521** [comment]
When the speed of unityexplorer is set to 0, the game is paused

**Comment by alan42521** [comment]
But now it could only be typed manully

**Comment by yukieiji** [comment]
I'm not quite sure what you mean by `Speed`...do you mean `MoveSpeed`?

**Comment by alan42521** [comment]
In other word, the time of the unity explorer

**Comment by alan42521** [comment]
![image](https://github.com/user-attachments/assets/f6bdb236-8402-42a8-8cda-585d8d250b96)

**Comment by yukieiji** [comment]
Oh, okay, the `Time:` part, right?

**Comment by alan42521** [comment]
Right, is there any way to add a hotkey to change the time between 1 and 0?

**Comment by yukieiji** [comment]
I don't think it will be implemented immediately, but it seems possible to set any speed to any hotkey.
Would it be OK to implement 5 hotkeys to change the timescale to 0, 1, half, double, any and lock?

**Comment by alan42521** [comment]
It would help

**Comment by originalnicodr** [comment]
If you need a reference on how to implement this I did it myself on my fork.

[TimeScaleWidget.cs](https://github.com/originalnicodr/CinematicUnityExplorer/blob/master/src/UI/Widgets/TimeScaleWidget.cs)
[KeyPress listener](https://github.com/originalnicodr/CinematicUnityExplorer/blob/master/src/ExplorerBehaviour.cs#L108)
